tests/test_add_connections.py

```
@@ -523,3 +523,58 @@ def test_extend_combined_connection():
+    else:
+        lst1 = CreateList(3)
+        lst2 = CreateList(4)
+        lst3 = CreateList(5)
+        lst1.run()
+        lst2.run()
+        lst3.run()
+
+        combined = lst2.outs + lst3.outs
+        result_value = lst1.outs + combined
+
+    assert result_value == [0, 1, 2] + [0, 1, 2, 3] + [0, 1, 2, 3, 4]
```

Ruff (0.12.2)

Consider `[0, 1, 2, 0, 1, 2, 3, 0, 1, 2, 3, 4]` instead of concatenation

Replace with `[0, 1, 2, 0, 1, 2, 3, 0, 1, 2, 3, 4]`

(RUF005)